src/Rules/PHPUnit/AssertEqualsIsDiscouragedRule.php

```diff
@@ -73,13 +73,7 @@ public function processNode(Node $node, Scope $scope): array
 					),
 				)->identifier('phpunit.assertEquals')
 					->fixNode($node, static function (CallLike $node) use ($correctName) {
-						if ($node instanceof Node\Expr\MethodCall) {
```

Comment:
since a recent change we have 
```php
if (!$node instanceof Node\Expr\MethodCall && ! $node instanceof Node\Expr\StaticCall) {
			return [];
}
```

at the very top of this rule, therefore we don't need to narrow here again anymore.